incubator/backup/README.md

```diff
@@ -2,31 +2,15 @@
 
 This chart starts a NuoDB database backup on existing NuoDB storage managers in a Kubernetes cluster using the Helm package manager.
 
-## TL;DR;
+## Command
 
 ```bash
-helm install nuodb/backup
+helm install nuodb/backup --name backup [--set parameter] [--values myvalues.yaml]
```

Comment:
we should not be documenting the use of Helm and all the possible helm parameters.

Reply:
We don't attempt to document these. These are two important ones they will likely find useful.